docs/app/Examples/elements/Rail/RailExamples.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react'
+import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample'
+import ExampleSection from 'docs/app/Components/ComponentDoc/ExampleSection'
+
+export default class RailExamples extends Component {
```

Comment:
As part of the coming doc site updates, we'll be using functional components for all stateless examples.  Could we update from this:

```js
import React, { Component } from 'react'

export default class RailExamples extends Component {
  render() {
    return (
      // ...
    )
  }
}
```

To this:

```js
import React from 'react'

const RailExamples = () => (
  // ...
)
export default RailExamples
```